### ISSUE TITLE
Codegen: Error when space in primary key name

### ISSUE THREAD

**Issue by cadam11** [MemberJunction/MJ#227]
In the create entity SP, we include a line to select the newly-created row to return. In that select line, the entity `CodeName` is used as the column name but the table will have `Name` as the column name. This results in an error for columns that don't naturally have SQL-valid names. E.g.

```
"Invalid column name 'Item_Number'."
````

The entity's PK, `Item Number`, gets `Item_number` as its `CodeName`. This gets rendered as

```sql
SELECT * FROM [test].[vwItems__test] WHERE [Item_Number] = @Item_Number
```

## Expected result

CodeGen should use the correct column name, i.e.

```sql
SELECT * FROM [test].[vwItems__test] WHERE [Item Number] = @Item_Number
```

**Comment by AN-BC** [comment]
@cadam11 found and fixed.

**Comment by cadam11** [comment]
> @cadam11 found and fixed.

If you can add `Fixes #227` in the commit message, it will link to, and close, this issue.
